lib/modules/blockchain_process/blockchain.js

```diff
@@ -93,22 +94,30 @@ Blockchain.prototype.initProxy = function() {
   this.config.wsPort += constants.blockchain.servicePortOnProxy;
 };
 
-Blockchain.prototype.setupProxy = function() {
+Blockchain.prototype.setupProxy = async function() {
   const proxy = require('./proxy');
   const Ipc = require('../../core/ipc');
 
-  let ipcObject = new Ipc({ipcRole: 'client'});
-  this.rpcProxy = proxy.serve(ipcObject, this.config.rpcHost, this.config.rpcPort, false);
-  this.wsProxy = proxy.serve(ipcObject, this.config.wsHost, this.config.wsPort, true);
+  if(!this.proxyIpc) this.proxyIpc = new Ipc({ipcRole: 'client'});
+  
+  let wsProxy;
+  if(this.config.wsRPC) {
```

Comment:
@alaibe is it conceivable that someone would have ws enabled but http disabled? Should we conditionally setup `rpcProxy` (http proxy) in the same manner?

Reply:
@iurimatias is there a config option like `config.wsRPC` that can be used to toggle http rpc on/off?  If not, should it be added?